snap7/util.py

```diff
@@ -94,6 +94,10 @@
 import logging
 from snap7 import six
 import re
+import numpy as np
```

Comment:
This is unused. Needs to be removed. We don't want more not-builtin dependencies if possible